packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -2242,8 +2242,12 @@ export interface IChoiceGroupOptionStyleProps {
     // (undocumented)
     hasImage?: boolean;
     // (undocumented)
+    imageHeight?: number;
+    // (undocumented)
     imageIsLarge?: boolean;
     // (undocumented)

```

Comment:
>// (undocumented) [](start = 4, length = 17)

Can you add some documentation here? Also, in that documentation, mention what would be the values, if they are supplied. especially because they are optional.

Reply:
Add docs